pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/create.yaml

```diff
@@ -27,7 +27,7 @@ spec:
   initialUser:
     user: postgres
     password:
-      value: postgres
```

Comment:
btw, I'm just curious about this change. Is there a specific reason to update the  `password.value` (like password requires to contain numbers)?

Reply:
For a user to enable the public IP feature, they'll also need to enforce password complexity as a security requirement. Hence, the password change